server/templates/place.html

```diff
@@ -17,7 +17,15 @@
 
 {% set main_id = 'dc-places' %}
 {% set page_id = 'page-dc-places' %}
-{% set title = place_name + ' ' + category + ' - ' + _('Place Explorer') %}
+{% set category_title = place_name + ' ' + category %}
+{% set overview_title = place_name + ' - Economics, Health, Equity, Crime, Education, Demographics, Housing, Environment' %}
```

Comment:
Not every place will have all of these categories (example: no crime here http://localhost:8080/place/geoId/06013 )

Reply:
can we pull these categories programmatically and translate them (both here and in the overview description below)

Reply:
Nit: move category_title and overview_title inline below in the if / else statement

Reply:
Moved the title logic into the if/else statement.

As discussed offline, pulling the category names is a larger change. The current logic lies downstream and needs to wait for all the data to load before calculating which categories are available. Leaving the current title as-is and title updates can be a future change.

Reply:
Update: added localization to the titles, while keeping the title format the same.